templates/TEMPLATE_terraform.tfvars

```diff
@@ -403,9 +403,36 @@ data_studio_container_version             = "0.7.0"
 flag_limit_data_studio_to_some_workspaces = false
 data_studio_eligible_workspaces           = ""
 
-data_studio_template_container_version_jupyter = "0.7.0"
-data_studio_template_container_version_rstudio = "0.7.0"
-data_studio_template_container_version_vscode = "0.7.0"
+# For full list of images Seqera makes available, please see: https://public.cr.seqera.io/
+# NOTE: Current as of v24.1.x, only images from publicly-available repositories can be used.
+#       Private repositories will be supported in a future iteration.
+data_studio_options = {
+      vscode1_83_0 = {
+            qualifier = "VSCODE-1-83-0"
+            icon = "vscode"
+            container = "public.cr.seqera.io/platform/data-studio-vscode:1.83.0-0.7"
+      },
+      jupyter4_1_5 = {
+            qualifier = "JUPYTER-4-1-5"
+            icon = "jupyter"
+            container = "public.cr.seqera.io/platform/data-studio-jupyter:4.1.5-0.7"
+      },
+      rstudio4_0_0 = {
+            qualifier = "RSTUDIO-4-0-0"
+            icon = "rstudio"
+            container = "public.cr.seqera.io/platform/data-studio-rstudio:4.0.0-0.7"
+      },
+      rstudio4_4_1 = {
+            qualifier = "RSTUDIO-4-4-1"
+            icon = "rstudio"
+            container = "public.cr.seqera.io/platform/data-studio-rstudio:4.4.1-0.7.1"
```

Comment:
For other images you use `major.minor` version but for this specific you use patch version. Why?

Reply:
Easy: I simply forgot to update them! Fixed now with a quick patch - thanks for the extra set of eyes on this! 👍

Reply:
Actually, I think that having `major.minor` might be more beneficial in the enterprise. When you set clients to 0.7 we can release a fixed patch version and there will be no need to update configuration. We won't be delivering any breaking changes between the patches.

Reply:
@weronikasosnowskaseqera FYI I chatted with @gwright99 and @schaluva and for GxP compliance it's best they use the `major.minor.patch` semantic versioning with the understanding that the customer will need to closely manage their installations of tower-connect. Customers will be advised of the increased operational overhead this incurs and may choose to reassess later.

Reply:
We are using `major.minor` in cloud and I think we should strongly advice to do the same for enterprise unless there is a strong argument for it. The changing image digests doesn't feel like a strong enough argument. AFAIK we already have changing digests whenever wave is involved, how does that play with GxP compliance?

As a small side note: if we end up suggesting `major.minor.patch` we probably should have a discussion about how to inform enterprise customers about path level increases and other implications (publishing release notes, etc)

Reply:
Field experience has taught me that leaving some things unstated / magic always ends up biting us later on via support. 

For the purposes of the CX Installer, I'm going to show how to go down to `major.minor.patch` (_and have this by default_) but will have a note that such level of control will introduce a heavy operational burden to:

1. Remain aware when new container images are present.
2. The need for (short) outages to ensure the updating container images can be made available via Data Studios.

I'll leave it to each implementing site to make their own decision if they want the control vs sticking with `major.minor` only.

Reply:
Don't get me wrong, it is a very fair concern. My point is more about having a company wide alignment about what we communicate/suggest which currently doesn't seem to be the case. In engineering we favour `major.minor` mainly because it allows us to ship bugfixes to existing data studio sessions, see [ADR](https://github.com/seqeralabs/adr/blob/master/data-studios/20240705-data-studios-client-images-versioning.md).

As a small side note: most probably, we will move to provide the default values in the application yml sooner or later in which case we will put in `major.minor` (after all it's what we use for cloud).

Reply:
+1 for company-wide alignment. 👍 

Until that happens, however, I'm favouring explicitness and control over convenience.

Reply:
Fair enough. Just beware that you are introducing misalignment with that :sweat_smile: we are already using `major.minor` for prod cloud deployment in engineering and don't plan to switch unless there is a strong argument for it :innocent: